quotefault_api/__init__.py

```diff
@@ -187,7 +187,8 @@ def newest():
     date = request.args.get('date')
     submitter = request.args.get('submitter')
     speaker = request.args.get('speaker')
-    query = query_builder(date, None, submitter, speaker).order_by(Quote.id.desc())
```

Comment:
Was this failing pylint? It seems weird to reformat this in an unrelated PR.

Reply:
the pylint failures were for the changes i made to the `str_to_datetime` function

Reply:
i believe the change in the line you are highlighting may have just been some auto-indentation thing trying to format the code when i saved it

Reply:
I'd rather this not get auto-formatted unnecessarily.